.github/workflows/build.yml

```
@@ -0,0 +1,22 @@
+name: Build Website
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy-website:
+    name: Build Website
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '2.7.0'
+        bundler-cache: true
+    - name: Run .github_actions.sh
+      run: NOKOGIRI_USE_SYSTEM_LIBRARIES=true ./.github_actions.sh
```

Add this variable to the enviroment variables.  Then you can change this line to:

```suggestion
      run: |
        bundle exec jekyll build --strict_front_matter
        bundle exec htmlproofer ./_site --check-html --only-4xx --http-status-ignore 429 --alt-ignore '/.*/'
```